using-mobile-extensions/places-extension/places-monitoring-extension/using-your-own-monitor.md

```diff
@@ -1,4 +1,142 @@
 # Using your own monitor
 
-This information is  in progress.
+You can also use your own monitoring services and intergrate with Places using the Places Extension API's
 
+## Registering Geofences
+
+In the occasion of using your own monitoring services, register the geofences of the near by point of intrests around the current location by implementing the following steps.
```

Comment:
Please change this sentence so that it reads as follows:

"If you decide to use your monitoring services, register the geofences of the POIs around your current location by completing the following steps:"